IronSkydive.html

```diff
@@ -0,0 +1,175 @@
+        <article class="col"><h4 class="member-name">Susan Phillps</h4><br><img class="col-img" src="https://s3-eu-west-1.amazonaws.com/ih-materials/uploads/upload_a18d6123a7c8e75f7e70a4e59b941093.jpg"alt="Picture of Susan"></article>
+        <article class="col"><h4 class="member-name">Taylor Roberts</h4><br><img class="col-img" src="https://s3-eu-west-1.amazonaws.com/ih-materials/uploads/upload_7104a331530d1b0611da55093b7dc421.jpg"alt="Picture of Taylor"></article>
+      </div>
+     </section>
+     <section id="schedule" class="container"> 
+       <h3> Schedule</h3>
+       <table class="schedule-table" cellpadding="0" cellspacing="0">
+        <thead>
+         <tr>
+           <th>Time</th>
+           <th>Monday</th>
+           <th>Tuesday</th>
+           <th>Wednesday</th>
+           <th>Thursday</th>
+           <th>Friday</th>
+           <th>Saturday</th>
+         </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>9:00 - 11:00</td>
+            <td></td>
+            <td></td>
+            <td>x</td>
+            <td></td>
+            <td>x</td>
+            <td>x</td>
+         </tr>
+         <tr>
+            <td>12:00 - 14:00</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>x</td>
+            <td>x</td>
+         </tr>
+          <tr>
+            <td>15:00 - 17:00</td>
+            <td></td>
+            <td></td>
+            <td>x</td>
+            <td>x</td>
+            <td>x</td>
+            <td>x</td>
+         </tr>
+       </tbody>
+       </table>
+       <h3> Schedule a Time Slot</h3>
+       
+       <form action="/register" method="POST" class="slot-form">
+         <div class="slot-form-container">
+           <label for="email">Email</label>
+           <input type="email" id="email" placeholder="Enter your email"/>
+         </div>
+         
+         <div class="slot-form-container">
+           <label for="participants"> Number of Participants</label>
+           <input type="number" id="participants" placeholder="Number of participants"/>
+         </div>
+         
+         <div class="slot-form-container">
+           <label for="date">Date</label>
+           <input type="date" id="date"/>
+         </div>
+         
+         <div class="slot-form-container">
+           <label for="time"> Time</label>
+           <select id="time">
+             <option value="early">9:00 - 11:00</option>
+             <option value="mid">12:00 - 14:00</option>
+             <option value="late">15:00 - 17:00</option>
+           </select>
+         </div>
+         
+         <div class="slot-form-container">
+           <button type="submit">Reserve Your Slot!</button>
+         </div>
+      </form>
+    </section>
+     <footer class="dark-background footer"> 
+       <div class"container">
```

Comment:
Your content is contained in the div not directly in the footer - you need to apply flexbox to that div in order for it to work.